src/Content/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,325 @@
+                            className={style.boardWorkSpaceSideBarItem}
+                            href="/"
+                          >
+                            <span
+                              className={
+                                style.boardWorkSpaceSideBarItemIconContainer
+                              }
+                            >
+                              <img
+                                src="https://img.icons8.com/external-becris-lineal-becris/16/000000/external-users-mintab-for-ios-becris-lineal-becris.png"
+                                alt=""
+                              />
+                            </span>
+                            <span>
+                              <span>Участники (1)</span>
+                            </span>
+                          </a>
+                          <a
+                            className={style.boardWorkSpaceSideBarItem}
+                            href="/"
+                          >
+                            <span
+                              className={
+                                style.boardWorkSpaceSideBarItemIconContainer
+                              }
+                            >
+                              <img
+                                src="https://img.icons8.com/windows/16/000000/settings--v1.png"
+                                alt=""
+                              />
+                            </span>
+                            <span>
+                              <span>Настройки</span>
+                            </span>
+                          </a>
+                          <div
+                            className={
+                              style.boardWorkSpaceUpgradeButtonContainer
+                            }
+                          >
+                            <button
+                              type="button"
+                              className={style.boardWorkSpaceUpgradeButton}
+                            >
+                              <div>
+                                <span
+                                  className={
+                                    style.boardWorkSpaceUpgradeButtonIconContainer
+                                  }
+                                >
+                                  <img
+                                    src="https://img.icons8.com/dusk/16/000000/upgrade.png"
+                                    alt=""
+                                  />
+                                </span>
+                              </div>
+                              <div>Повысить</div>
+                            </button>
+                          </div>
+                        </div>
+                      </div>
+                    </div>
+                  </div>
+                  <Boards />
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
```

Comment:
This indention gives me a stroke. You are better off with splitting it into smaller components.